packages/tradershub/src/components/CurrencySwitcher/CurrencySwitcher.tsx

```diff
@@ -89,11 +76,20 @@ const CurrencySwitcher = () => {
                     onClick={() => {
                         show(
                             <Modal>
-                                <Modal.Header title='Select account' titleClassName='text-typography-default' />
-                                <Modal.Content className='overflow-y-scroll'>
+                                <Modal.Header title='Select account' titleClassName='text-[14px] sm:text-[16px]' />
+                                <Modal.Content>
```

Comment:
@aizad-deriv Why did you remove this scroll class? if the number of currencies will be more than the height of the modal, it can not be scrolled without this.

Reply:
Because on ModalContent component we already have `overflow-y-auto`

Reply:
**
![image](https://github.com/binary-com/deriv-app/assets/103104395/12a63883-7420-4696-a17e-1e2bf7d5b82a)
**

Reply:
I see 👍